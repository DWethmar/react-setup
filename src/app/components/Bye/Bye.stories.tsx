import React from 'react';
import { Bye } from './Bye';

export default {
    title: 'Bye',
    component: Bye,
};

export const Default = (): React.ReactElement => <Bye name="programmer" />;
