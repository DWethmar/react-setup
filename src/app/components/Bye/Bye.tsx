import * as React from 'react';
import * as PropTypes from 'prop-types';

import './Bye.scss';

export interface ByeProps {
    name: string;
}

const Bye: React.StatelessComponent<ByeProps> = ({ name }) => {
    return <h1>Bye {name}!</h1>;
};

Bye.propTypes = {
    name: PropTypes.string.isRequired,
};

export { Bye };
