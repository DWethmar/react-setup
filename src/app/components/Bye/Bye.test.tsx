import React from 'react';
import { shallow } from 'enzyme';
import { Bye } from './Bye';

describe('Bye', () => {
    it('should match snapshot', () => {
        const component = shallow(<Bye name="programmer" />);

        expect(component).toMatchSnapshot();
    });
});
