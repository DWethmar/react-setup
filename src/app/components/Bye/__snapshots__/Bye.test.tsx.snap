// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bye should match snapshot 1`] = `
<h1>
  Bye 
  programmer
  !
</h1>
`;
